src/app.rs

```diff
@@ -178,9 +180,18 @@ impl Application {
                 }
                 config.healthchecks.set_require_healthy(require_healthy);
 
-                #[cfg(feature = "datadog-pipelines")]
+                #[cfg(feature = "enterprise")]
```

Comment:
Noting the feature rename internally to `enterprise`, to better accommodate the overarching vision of our integration with DD.